### PR TITLE
Project Details Draft

### PR DESCRIPTION
**Pull request by Faro0q**
Rough draft for project details:
Added required details in the project boxes.

### CODE REVIEW

File: proposal.html [hunk 1]
```diff
@@ -14,30 +14,50 @@
...
                 </div>
                 <div class="mid-box">
-                    mid-box-right
+                    People who love to share their thoughts with others<br>
+                    <br>
+                    Goal 1: Have a user-friendly UI<br>
+                    Goal 2: Have a fully polished website by the end of the semester<br>
+                    <br>
+                    We selected this topic because we think this will really challenge us to
+                    the fullest and can keep adding complexities as we go.
```
Comment (wcygan) [inline proposal.html]: Can you add the section headers above the answers?

E.g., 

**Question**:
Answer

File: proposal.html [hunk 2]
```diff
@@ -14,30 +14,50 @@
...
                 </div>
             </div>
         </div>
         <div class="row">
             <div class="footer-container">
                 <div class="footer-items">
                     <div class="footer-box">
-                        footer-box-left
+                        - Link to API<br>
+                        - Twitter.com<br>
```
Comment (wcygan) [inline proposal.html]: Why are we using the [Twitter API](https://developer.twitter.com/en/docs)? How are we going to use it?

File: proposal.html [hunk 3]
```diff
@@ -14,30 +14,50 @@
...
                     <div class="footer-box">
-                        footer-box-left
+                        Questions:
                     </div>
                     <div class="footer-box">
-                        footer-box-left
+                        Accessibility implementation thoughts:<br>
+                        Maybe text to speech?<br>
+                        Light/Dark mode?<br>
+                        Font change button?<br>
```
Comment (wcygan) [inline proposal.html]: Can you add section headers here too?

- **Resource Links**:
- **Questions + Concerns**:
- **Accessibility Implementation**:

**Review by wcygan — CHANGES_REQUESTED** [review]
It's almost there -- needs some polish before we can merge

File: proposal.html (hunk 2 above)
Comment (Faro0q) [reply proposal.html]: We are not using twitter API. Twitter.com falls underneath the question: Links to site you would like to model your site after. I will update this with questions and answers to eliminate confusion

Comment (wcygan) [reply proposal.html]: sounds good 🤠

**Review by wcygan — APPROVED** [review]
yolo

**Pull request merged**